src/matrix/room/timeline/entries/BaseEventEntry.js

```diff
@@ -53,10 +53,13 @@ export class BaseEventEntry extends BaseEntry {
     }
 
     /**
-        aggregates local relation or local redaction of remote relation.
+        Aggregates relation or redaction of remote relation.  
+        Used in two situations:
+        - to aggregate local relation/redaction of remote relation
+        - to mark this entry as being redacted in Timeline._updateEntriesFetchedFromHomeserver
         @return [string] returns the name of the field that has changed, if any
     */
-    addLocalRelation(entry) {
+    addRelation(entry) {
```

Comment:
The comment above is great, thanks, but not sure about this name change. It gives the impression that this is the only way relations are tracked, while 99% of the time, relations are tracked by RelationWriter, and this is just for some "edge" cases like local echo and when fetching something from /context.

If we can't come up with a better name (I'm not able to come up with one), I'd keep `addLocalRelation` tbh.